Tools/Pipeline/Common/PipelineController.cs

```diff
@@ -515,6 +515,8 @@ private void DoBuild(string commands)
                 _buildProcess.StartInfo.StandardOutputEncoding = encoding;
                 _buildProcess.OutputDataReceived += (sender, args) => View.OutputAppend(args.Data);
 
+                View.OutputAppend(string.Format("Working Directory: {0} ", _buildProcess.StartInfo.WorkingDirectory));
+                View.OutputAppend(string.Format("Running: {0} {1}", _buildProcess.StartInfo.FileName, _buildProcess.StartInfo.Arguments));
```

Comment:
Is there a concern the working directory and launch EXE could be bad?

Reply:
nope, I thought it might be useful information to report.. I can take that out